rust-toolchain

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.70.0"
+channel = "nightly"
```

Comment:
This may be problematic. Nightly famously breaks backwards compatibility, so it may inadvertently break main. There's probably a way to pin this to a specific date.

Reply:
This is indeed problematic, @entropidelic is right now working on removing this requirement by making `cairo_native` switch to stable